assignment/scripts/3-music-collection.js

```diff
@@ -1 +1,43 @@
-console.log('***** Music Collection *****')
+console.log("***** Music Collection *****");
+collection = [];
+
+function addToCollection(title, artist, yearPublished) {
+  let tempCollection = {
+    title: `${title}`,
+    artist: `${artist}`,
+    yearPublished: `${yearPublished}`,
+  };
+  collection.push(tempCollection);
+  console.log(collection);
```

Comment:
Careful! The assignment called for you to _return the new album_, not to log the collection.

```suggestion
  return tempCollection;
```

Remember that a `console.log` doesn't actually **do** anything as far as JavaScript is concerned. It takes some value and prints it out so _you the developer_ can see it, but it won't affect how your code actually runs. If you want to get a variable like the new album out of a function so the rest of our code can use (which the requirements specify here) , you _must_ return it.